client/README.md

```diff
@@ -0,0 +1,70 @@
+# Getting Started with Create React App
```

Comment:
Remove the default generated readme details by create-react-app, add details in readme about the components you are going to create, add about installation process.

Reply:
Sure, I will do it in next PR.